src/components/AddOffering/AddOffering.js

```diff
@@ -172,7 +172,7 @@ class AddOffering extends React.Component {
         this.state.comment,
         this.state.addedItems,
         this.props.currentUser,
-        this.props.itemDetail.id,
+        this.props.itemDetail.id
```

Comment:
this will likely need to be changed to `this.props.itemDetail._id`

Reply:
ooooh good point. Will change